### DISCUSSION TITLE
Multiple Simultaneous Outputs?

### DISCUSSION THREAD

**Discussion by website-user** [kyleneideck/BackgroundMusic#655] [Ideas]
Hey Kyle,

Thank you for making and maintaining this app!

Have / would you considered adding the ability to specify different (simultaneous) outputs, for different apps? 

This way you could, for example, play music on a set of speakers for all to hear, while limiting other audio sources to just your laptop speakers or headphones.

SoundSource from Rogue Amoeba, a similar app, has this functionality but is prohibitively expensive.

Thanks,
Liam

**Comment by augustodelucena** [comment]
Have you found any viable alternative for SoundSource?

**Reply by website-user** [reply]
Nope - but I did find that you can go into Settings--> Sound and set **Play sound effects through** _select output_, which was... almost good enough for my use case. 

Good luck,
